### ISSUE TITLE
npm package is missing poll.js (Version 3.7.1)

### ISSUE THREAD

**Issue by benurb** [cujojs/when#424]
Good morning,

like the title says the currently published package of when 3.7.1 on npmjs.org is missing at least the poll.js file. I can't say for sure if that's the only missing file.

Regards
Benjamin

**Comment by briancavalier** [comment]
Thanks @benurb.  That's very strange indeed, but sure enough, poll.js is not there :(  It's kind of hard to imagine how that could happen, other than a bug in npm.  I checked 3.7.0 (`npm install when@3.7.0`) as well as installing from my local development dir (`npm install path/to/when`).  Both of those worked correctly, ie node_modules/when/poll.js was installed in both cases.  AFAICT, neither package.json nor .npmignore has changed in a way that would exclude poll.js in 3.7.1.

I guess we could just try publishing a 3.7.2 and see what happens.

**Comment by benurb** [comment]
Yeah, sounds like a plan. It's really strange ...

**Comment by briancavalier** [comment]
Just checked `bower install when`, and that includes bower_components/when/poll.js.  Seems pretty convincing that it was some sort of npm publish issue.  I'll publish 3.7.2 in a few mins, fingers crossed.

**Comment by briancavalier** [comment]
I ran `npm pack` just before publishing 3.7.2, to verify that poll.js gets included, and it did:

```
❯ npm pack
when-3.7.2.tgz
❯ tar -tvzf when-3.7.2.tgz
-rw-r--r--  0 501    20       2348 Jan 30 10:42 package/package.json
-rw-r--r--  0 501    20       3540 Aug 20 13:49 package/README.md
-rw-r--r--  0 501    20       3172 Nov  2 15:40 package/poll.js
-rw-r--r--  0 501    20       1508 Oct 28 07:25 package/pipeline.js
...
```

Onward to `npm publish`

**Comment by briancavalier** [comment]
Published 3.7.2, and immediately did a test install, which did include poll.js.  Thanks again @benurb, and please let me know if installing 3.7.2 works for you.

**Comment by benurb** [comment]
Yep, working fine now. Thanks for your work :+1:
